java/server/src/org/openqa/selenium/remote/server/rest/ResultConfig.java

```diff
@@ -62,6 +59,9 @@ public ResultConfig(
     this.log = log;
     this.sessions = sessions;
     this.handlerFactory = getHandlerFactory(handlerClazz);
+
+    String hookEndPoint = System.getProperty("hookEndPoint", "http://localhost:8888");
+    this.wptHookAwareHandler = new WptHookAwareHandler(log, hookEndPoint);
```

Comment:
I am not sure about how the server is architected, but I wonder if we end up repeatedly creating new objects? If so, we should consider having a singleton or some such.